### ISSUE TITLE
Copy (only) the free resources from the "Learn JavaScript" post on JavaScript Scene

### ISSUE THREAD

**Issue by ericelliott** [jshomes/learning-resources#2]
* [ ] Add to the [JavaScript](https://github.com/jshomes/learning-resources#javascript) track.
* [Learn JavaScript on Medium](https://medium.com/javascript-scene/learn-javascript-b631a4af11f2)

**Comment by chrisrhoton** [comment]
@ericelliott On it!

**Comment by ericelliott** [comment]
:+1:

**Comment by ericelliott** [comment]
I just made an index to organize this project better. Please check it out and let me know what you think. =)

**Comment by chrisrhoton** [comment]
Looks great. Definitely like this format better. Regret delays from my end. Currently on vacation and only have intermittent Internet access. You'll see a new pull request from me as soon as my wifi cooperates.

**Comment by FlyDuo** [comment]
Yo, I've got 2 new resources for ya, hope this helps:

- http://gymnasium.aquent.com/courses/GYM/104/0/about (Free online courses geared towards getting you ready for working in the field)
- http://gymnasium.aquent.com/courses/GYM/002/0/about (Node.js course, also Free.)

BTW: Eric, I'm a web developer, have been since '01, and js been that beast in my mind not conquered. At this point, not having real js chops hasn't been helpful (to say the least), so I'm learning and working really hard with the intention over time to build some deep JS skillz, and your work has been really helpful. And your articles encouraging. I've learned things over the years many, many times, but have always been intimidated by js that's anything more than pretty simple "one liners", this go round I'm trying to dedicate more time to it as well as have something in mind to build. I'ma take your recommendation (from your article) and try my hand at creating a simple game to get them skillz locked in. Thank you so much for helping people like me develop deeper code skillz.
~ Reuben

**Comment by ericelliott** [comment]
Thanks. Do you know how to create a pull request?

**Comment by FlyDuo** [comment]
Yes, I've done pull requests using BitBuckets/Jira tools for projects on jobs, but I'm not sure how I should do things in this particular situation - I'm reading the "contributing guide" now for direction. If you don't mind guiding me if I have questions, I'd love to be of service.

**Comment by FlyDuo** [comment]
I've read the contributing docs, cloned that repo, and read the article on the JSHomes mission, [JSHomes Platform Documentation] (https://github.com/jshomes/JSHomes-Platform/tree/master/docs) and  [Courses] (https://github.com/jshomes/JSHomes-Platform/blob/master/docs/future.md), I'm not sure what I should be doin next (if anything?). I'm sorry if I'm adding more work to you to answer questions, etc., I just want to make sure I'm doing things correctly. 

This is my first time doing this type of thing. (I use git regularly, but not in this way and never Vagrant.), I'm working on getting Vagrant setup, ran into some permissions issues with the Vagrant init command on Windows; I'll try to make some time hopefully later this week to get that goin.

**Comment by ericelliott** [comment]
No worries. Just edit the relevant file, add your contributions, and submit the PR.
